Bebb-Protocol-and-Apps/PWS
# Enhance default spaces

This PR updates the Default Spaces section on the Create page;

it adds an environment to the existing default space
and adds two new default spaces to choose from:
a space-themed one
and an island-themed one

To test:
on the Create page, make sure that all three default space options are displayed correctly (incl. the preview)
ensure that each default space can be created
each created space should load as expected
other functionality should not be affected
